src/ripple/core/impl/LoadEvent.cpp

```diff
@@ -29,73 +30,74 @@ LoadEvent::LoadEvent (
         LoadMonitor& monitor,
         std::string const& name,
         bool shouldStart)
-    : m_loadMonitor (monitor)
-    , m_isRunning (shouldStart)
-    , m_name (name)
-    , interval_ { std::chrono::steady_clock::now() }
+    : monitor_ (monitor)
+    , running_ (shouldStart)
+    , name_ (name)
+    , mark_ { std::chrono::steady_clock::now() }
     , timeWaiting_ {}
     , timeRunning_ {}
 {
 }
 
 LoadEvent::~LoadEvent ()
 {
-    if (m_isRunning)
+    if (running_)
         stop ();
 }
 
 std::string const& LoadEvent::name () const
 {
-    return m_name;
+    return name_;
 }
 
 std::chrono::milliseconds
-LoadEvent::getSecondsWaiting() const
+LoadEvent::waitTime() const
 {
-    return timeWaiting_;
+    // C++17: std::chrono::ceil
+    return ceil<std::chrono::milliseconds>(timeWaiting_);
```

Comment:
I have no objection to this code, but I do feel I should point out that it is still a minor change in logic from the way it is was.  Previously latency in the range [0ms, 1ms] was clipped to 0ms.  Now latency in the range (0ms, 1ms] is pushed up to 1ms.